### PR TITLE
java generated models now initialize lists

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: ds3-autogen-java/src/main/resources/tmpls/response/get_job_chunks_ready_response_template.ftl [hunk 1]
```diff
@@ -26,7 +26,7 @@ ${javaHelper.createAllResponseResultClassVars(responseCodes)}
             case 200:
                 try (final InputStream content = webResponse.getResponseStream()) {
                     this.jobWithChunksApiBeanResult = XmlOutput.fromXml(content, JobWithChunksApiBean.class);
-                    if (this.jobWithChunksApiBeanResult.getObjects() == null) {
+                    if (this.jobWithChunksApiBeanResult.getObjects().size() == 0) {
```
Comment (shabtaisharon) [inline ds3-autogen-java/src/main/resources/tmpls/response/get_job_chunks_ready_response_template.ftl]: is there an option to use a Guard method here?

**Comment by shabtaisharon** [comment]
This branch is out-of-date

File: ds3-autogen-java/src/main/resources/tmpls/response/get_job_chunks_ready_response_template.ftl (hunk 1 above)
Comment (rpmoore) [inline ds3-autogen-java/src/main/resources/tmpls/response/get_job_chunks_ready_response_template.ftl]: Guard idea is not a bad idea.  I think there is a Guard class in the sdk.

**Comment by RachelTucker** [comment]
Updated

**Comment by RachelTucker** [comment]
Updated

**Pull request merged**